mongodb.html.md.erb

```diff
@@ -40,3 +40,29 @@ To connect to a running MongoDB instance with your local development tools, you
 ## <a id='sample-application'></a> Sample Application
 
 Cloud Foundry: [Spring Music Example](https://github.com/cloudfoundry-samples/spring-music)
+
+## <a id='specifications'></a> Specifications
+
+Our [MongoDB](https://www.mongodb.com/community) offering consists of 3 Kubernetes containers configured as a replica set.
+
+## <a id='roles'></a> Roles
+
+You will receive the role `clusterMonitor` on admin db , `readWrite` and `dbAdmin` roles on your DB.
```

Comment:
Must be "You will receive the role `clusterMonitor` on the admin DB, and the roles `readWrite` and `dbAdmin` on your DB."